base_external_dbsource/models/base_external_dbsource.py

```diff
@@ -172,6 +171,48 @@ def execute(
         else:
             return rows
 
+    @api.onchange('connector')
+    def _change_placeholder_str_connect(self):
+        """
+        Changing value of connector field will automatically
+        set value in conn_string field.
+        """
+        if self.connector == 'fdb':
```

Comment:
IMO these should all be in the independent modules & use an interface strategy in the core similar to what I developed for the others. The core module should only have a concept of postgres, and that should still be in an independent method. This avoids the giant unmanageable if chains, in favor of a more flat approach - which jives with [PEP-20](https://www.python.org/dev/peps/pep-0020/) better IMO.

I even made a helper method, example:
* https://github.com/multidadosti-erp/server-tools/blob/b231d610b5b0de3e8c5e9d0c63156e6d50899247/base_external_dbsource/models/base_external_dbsource.py#L103

Then the corresponding methods that it actually calls:
* https://github.com/multidadosti-erp/server-tools/blob/b231d610b5b0de3e8c5e9d0c63156e6d50899247/base_external_dbsource/models/base_external_dbsource.py#L336
* https://github.com/multidadosti-erp/server-tools/blob/b231d610b5b0de3e8c5e9d0c63156e6d50899247/base_external_dbsource_firebird/models/base_external_dbsource.py#L36

High level info on this interface/adapter implementation is in the [Class level doc block](https://github.com/multidadosti-erp/server-tools/blob/b231d610b5b0de3e8c5e9d0c63156e6d50899247/base_external_dbsource/models/base_external_dbsource.py#L21).